app/main/views/frameworks.py

```diff
@@ -109,6 +119,55 @@ def update_framework(framework_slug):
             enforce_required=True
         )
 
+    if json_payload['frameworks'].get('status', None) == 'pending' and framework.status != 'pending':
```

Comment:
Minor thing, but I think `.get()` will return `None` anyway, so no need to set it as a default return value here.

Reply:
You're right, I just preferred to be explicit. It is a tad redundant though :man_shrugging:

Reply:
Yeah, fine either way.